### PR TITLE
Docs: Improve ease of docs on templating labels and annotations

### PR DESCRIPTION
**Pull request by grobinson-grafana**
**What is this feature?**

A couple of small improvements to the docs on templating labels and annotations that should make it more approachable and understandable for users.

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ weight: 117
 
 In Grafana it is possible to template labels and annotations just like in Prometheus. Those who have used Prometheus before should be familiar with `$labels` and `$value` as these variables contain the labels and value of the alert. You can use the same variables in Grafana to template labels and annotations, even if the alert does not use a Prometheus datasource.
 
-For example, suppose you want to create an alert rule in Grafana that fires when one of your instances is down for more than 5 minutes, and that each alert fired should have a summary annotation to tell you which instance is down:
+## Printing labels
+
+Suppose you want to create an alert rule that fires when an instance is down for more than 5 minutes. You want the alert to contain a summary telling you which instance is down. You can do this by creating a summary annotation that prints the instance label and the message "has been down for more than 5 minutes".
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:19]:
```suggestion
If you want to create an alert rule that fires when an instance is down for more than 5 minutes and you want the alert to contain a summary telling you which instance is down, create a summary annotation that prints the instance label and the message "has been down for more than 5 minutes".

```

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ weight: 117
 
 In Grafana it is possible to template labels and annotations just like in Prometheus. Those who have used Prometheus before should be familiar with `$labels` and `$value` as these variables contain the labels and value of the alert. You can use the same variables in Grafana to template labels and annotations, even if the alert does not use a Prometheus datasource.
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:15]:
```suggestion
In Grafana, you can template labels and annotations just like in Prometheus. Those who have used Prometheus before should be familiar with `$labels` and `$value` as these variables contain the labels and value of the alert. You can use the same variables in Grafana to template labels and annotations, even if the alert does not use a Prometheus data source.

```

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 3]
```diff
@@ -13,7 +13,9 @@ weight: 117
...
 
-For example, suppose you want to create an alert rule in Grafana that fires when one of your instances is down for more than 5 minutes, and that each alert fired should have a summary annotation to tell you which instance is down:
+## Printing labels
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:16]:
```suggestion
## Print labels
```

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 4]
```diff
@@ -33,65 +35,43 @@ This is because it is attempting to use a non-existing field `name` in `$labels.
 Instance {{ index $labels "instance.name" }} has been down for more than 5 minutes
 ```
 
-## Use values in labels and annotations
+## Printing the ValueString
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:38]:
```suggestion
## Print the ValueString
```

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 5]
```diff
@@ -33,65 +35,43 @@ This is because it is attempting to use a non-existing field `name` in `$labels.
...
 
-Grafana supports `$value` when templating labels and annotations. However, while `$value` in Prometheus is a floating point number contains the value of the expression, `$value` in Grafana is a string containing the labels and values of all Threshold, Reduce and Maths expressions. It does not contain the value of queries as a single query can return anywhere from 1 to 10,000s of rows or metrics.
+You can print the value of the alert with `$value`. However, where `$value` in Prometheus is a floating point number containing the value of the expression, in Grafana it is is a string containing the labels and values of all Threshold, Reduce and Maths expressions from the alert rule definition. It does not contain time series or tabular data, as a single query can return anywhere from 1 to 10,000s of rows or metrics.
 
-This `$value` variable is called the Value String. If you were to use it in the template of a summary annotation:
+This `$value` variable is called the ValueString. If you were to use it in the template of a summary annotation:
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:42]:
```suggestion
This `$value` variable is called the ValueString. If you use it in the template of a summary annotation:
```

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 6]
```diff
@@ -33,65 +35,43 @@ This is because it is attempting to use a non-existing field `name` in `$labels.
...
 
 ```
 {{ $labels.instance }} has an average 95th percentile request latency above 1s: {{ $value }})
 ```
 
-you will get the following summary:
+you would get the following summary:
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:48]:
```suggestion
you get the following summary:
```

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 7]
```diff
@@ -33,65 +35,43 @@ This is because it is attempting to use a non-existing field `name` in `$labels.
...
 
 ```
 http_server has an average 95th percentile request latency above 1s: [ var='B' labels={instance=http_server} value=10 ]
 ```
 
-To get just the value of `B` you can instead use `$values`:
+## Printing the values of individual expressions
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:54]:
```suggestion
## Print the values of individual expressions
```

File: docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md [hunk 8]
```diff
@@ -33,65 +35,43 @@ This is because it is attempting to use a non-existing field `name` in `$labels.
...
+
+You can also print the values of individual expressions with `$values`. For example, if you have an alert rule with a Reduce expression `B` and a Math expression `C`, you can print just the value of `B`:
 
 ```
 {{ $labels.instance }} has an average 95th percentile request latency above 1s: {{ $values.B }}
 ```
 
-and then you will get this summary:
+and then you would get this summary:
```
Comment (brendamuir) [inline docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md:62]:
```suggestion
Then you get this summary:
```

**Comment by grobinson-grafana** [comment]
Closed in favour of https://github.com/grafana/grafana/pull/63539

**Pull request closed without merging**